core/CMakeLists.txt

```diff
@@ -385,8 +385,6 @@ set(LOCALEDIR \"${CMAKE_INSTALL_LOCALEDIR}\")
 
 set(HAVE_CRYPTO 1)
 
-include(BareosTypeSizes)
```

Comment:
This whole commit is one of the coolest I have seen on this code. ;-) Very good, now we have stdint.h.